sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -926,6 +1176,25 @@ public partial class RecognizePiiEntitiesResultCollection : System.Collections.O
         public Azure.AI.TextAnalytics.TextDocumentBatchStatistics Statistics { get { throw null; } }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct RelativeTo : System.IEquatable<Azure.AI.TextAnalytics.RelativeTo>
```

Comment:
I think the name `RelativeTo` may be a bit generic. We should consider renaming it later to avoid conflicts with other libraries.

Reply:
I agree! I have this one in my notes to be discussed in a future arch review.